File: app/src/main/java/com/stochanskyi/itunescharts/data/DB/AppDatabase.java
package com.stochanskyi.itunescharts.data.DB;

import androidx.room.Database;
import androidx.room.RoomDatabase;

import com.stochanskyi.itunescharts.data.network.features.audiobooks.models.AudiobookModel;
import com.stochanskyi.itunescharts.data.network.features.movies.models.MovieModel;
import com.stochanskyi.itunescharts.data.network.features.podcasts.models.PodcastModel;

@Database(entities = {AudiobookModel.class, MovieModel.class, PodcastModel.class}, version = 1)
public abstract class AppDatabase extends RoomDatabase {
    public abstract DBDao dbDao();
}

File: app/src/main/java/com/stochanskyi/itunescharts/data/database/common/AppDatabase.kt
package com.stochanskyi.itunescharts.data.database.common

import androidx.room.Database
import androidx.room.RoomDatabase
import com.stochanskyi.itunescharts.data.database.favourites.FavouritesDao

@Database(entities = [FavouritesDao::class], version = 1)
abstract class AppDatabase: RoomDatabase() {

    abstract val favouritesDao: FavouritesDao

}